src/components/stats-plus.js

```diff
@@ -123,8 +123,8 @@ AFRAME.registerComponent("stats-plus", {
     this.el.setAttribute(this.name, false);
   },
   remove() {
-    this.el.sceneEl.removeListener("enter-vr", this.hide);
-    this.el.sceneEl.removeListener("exit-vr", this.show);
+    this.el.sceneEl.removeEventListener("enter-vr", this.hide);
```

Comment:
is it ever possible with a race condition for `sceneEl` to not be existent but this component method gets called in a delay?

Reply:
Not as far as I know. At least not in usual operation. A-Frame lifecycle should be synchronous and it should be the only thing that calls remove() directly.